pilot/pkg/model/service.go

```diff
@@ -233,12 +255,18 @@ func (p Protocol) IsTCP() bool {
 //  --> 172.16.0.1:54546 (with ServicePort pointing to 80) and
 //  --> 172.16.0.1:33333 (with ServicePort pointing to 8080)
 type NetworkEndpoint struct {
-	// Address of the network endpoint, typically an IPv4 address
+	// Family indicates what type of endpoint, such as TCP or Unix Domain Socket.
+	Family AddressFamily
```

Comment:
remove? I dont see this being used well

Reply:
It's used in `GetNetworkEndpointAddress()`